osclib/accept_command.py

```diff
@@ -140,8 +194,32 @@ def accept_other_new(self):
             self.create_new_links(self.api.project, req['packages'][0], oldspecs)
             changed = True
 
+        if self.api.delreq_monitor:
+            rqlist = self.find_virtual_accepted_requests(self.api.project)
+            for req in rqlist:
+                if self.virtual_accept_request_has_no_binary(self.api.project, req['packages'][0]):
+                    print 'Accepting request %d: %s' % (req['id'], ','.join(req['packages']))
+                    # Accepting delreq-monitor review
+                    self.api.do_change_review_state(req['id'], 'accepted',
+                                                    by_group=self.api.delreq_monitor,
+                                                    message='Virtual accepted to delete {}'.format(req['packages'][0]))
```

Comment:
nit: same wording change as above. `s/Virtual/Virtually/`, `s/to //`

Any reason to list the packages in the review state change as I would expect that to be clear given that is placed on the request? The rest of the text would seem sufficient.